app/components/markdown/markdown.js

```diff
@@ -222,11 +191,17 @@ export default class Markdown extends PureComponent {
     }
 
     renderHeading = ({children, level}) => {
-        const style = getStyleSheet(this.props.theme);
-
+        let style = getStyleSheet(this.props.theme);
+        style = {
+            ...style.block,
+            ...this.props.blockStyles[`heading${level}`]
+        };
+        const textStyle = {
+            paddingBottom: StyleSheet.flatten(this.props.textStyles[`heading${level}`]).lineHeight * 0.3
```

Comment:
Couldn't this be included in the block style instead of having to flatten it and calculate it after the fact?